Formula/exodriver.rb

```diff
@@ -18,7 +18,7 @@ class Exodriver < Formula
 
   def install
     system "make", "-C", "liblabjackusb", "install",
-           "HEADER_DESTINATION=#{include}", "DESTINATION=#{lib}"
+           "PREFIX=#{prefix}", "RUN_LDCONFIG=0", "LINK_SO=1"
```

Comment:
`DESTINATION = $(PREFIX)/lib` and `HEADER_DESTINATION = $(PREFIX)/include`

macOS defaults: `RUN_LDCONFIG ?= 0` and `LINK_SO ?= 1`
Linux defaults: `RUN_LDCONFIG ?= 1` and `LINK_SO ?= 0`